libhb/common.c

```diff
@@ -1432,8 +1438,8 @@ void hb_video_quality_get_limits(uint32_t codec, float *low, float *high,
         case HB_VCODEC_FFMPEG_VP8:
         case HB_VCODEC_FFMPEG_VP9:
         case HB_VCODEC_FFMPEG_VP9_10BIT:
-        case HB_VCODEC_SVT_AV1:
```

Comment:
I think those two HB_VCODEC_SVT_AV1* cases shouldn't be removed.

Reply:
I've looked up https://docs.nvidia.com/video-technologies/video-codec-sdk/nvenc-video-encoder-api-prog-guide/index.html and 0-51.0 should with fractional values supported should still be accurate for AV1